src/Compilers/VisualBasic/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1016,6 +1016,7 @@
         <Field Name="LocalSymbol" Type="LocalSymbol"/>
         <Field Name="InitializerOpt" Type="BoundExpression" Null="allow"/>
         <Field Name="InitializedByAsNew" Type="Boolean" />
+        <Field Name="ArrayCreationOpt" Type="BoundArrayCreation" Null="allow"/>
```

Comment:
><Field Name="ArrayCreationOpt" Type="BoundArrayCreation" Null="allow"/> [](start = 8, length = 71)

Could you please add an assert into Validate method for this node that  ```InitializerOpt IsNot ArrayCreationOpt```?

Reply:
Done.

---
In reply to: [146959520](https://github.com/dotnet/roslyn/pull/22842#discussion_r146959520) [](ancestors = 146959520)